src/main/java/org/profit/telegrambot/util/InlineKeyboardUtil.java

```diff
@@ -0,0 +1,190 @@
+        button.setCallbackData(data);
+        return button;
+    }
+
+    private static List<InlineKeyboardButton> getRow(InlineKeyboardButton... buttons) {
+        return Arrays.asList(buttons);
+    }
+
+    private static List<List<InlineKeyboardButton>> getRowList(List<InlineKeyboardButton>... rows) {
+        return Arrays.asList(rows);
+    }
+
+    public static InlineKeyboardMarkup categoryInlineMarkup() {
+
+        CategoryService.loadCategoryList();
+
+        List<List<InlineKeyboardButton>> rowList = new ArrayList<>();
+        for (Category category : Database.categoryList) {
+            List<InlineKeyboardButton> buttonList = new ArrayList<>();
+            InlineKeyboardButton button = new InlineKeyboardButton(category.getName());
+            button.setCallbackData("add_product_category_id/" + category.getId());
+            buttonList.add(button);
+            rowList.add(buttonList);
+        }
+        return new InlineKeyboardMarkup(rowList);
+    }
+
+    public static InlineKeyboardMarkup categoryShow() {
+        CategoryService.loadCategoryList();
+
+        List<List<InlineKeyboardButton>> listList = new ArrayList<>();
+        for (Category category : Database.categoryList) {
+            List<InlineKeyboardButton> buttonList = new ArrayList<>();
+            InlineKeyboardButton button = new InlineKeyboardButton(category.getName());
+            button.setCallbackData("show_category/" + category.getId());
+            buttonList.add(button);
+            listList.add(buttonList);
+        }
+
+        InlineKeyboardButton button = getButton("back", "back_from_category_list");
+        List<InlineKeyboardButton> row = getRow(button);
+        listList.add(row);
+
+        return new InlineKeyboardMarkup(listList);
+    }
+
+
+    public static InlineKeyboardMarkup Menu() {
+
+        InlineKeyboardButton menuButton = getButton("📠MENU", "menu_");
+        InlineKeyboardButton magazineButton = getButton("🏪Our address", "shop_url");
+        InlineKeyboardButton contactButton = getButton("📞Our social networks", "web_network");
+        InlineKeyboardButton helpButton = getButton("💬HELP", "help");
+
+        magazineButton.setUrl("https://www.google.com/maps/place/%D0%A6%D0%B5%D0%BD%D1%82%D1%80%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9+%D1%83%D0%BD%D0%B8%D0%B2%D0%B5%D1%80%D0%BC%D0%B0%D0%B3/@50.9067489,34.7966222,15z/data=!4m5!3m4!1s0x0:0xdc737b63790a2531!8m2!3d50.9067489!4d34.7966222");
+        helpButton.setUrl("https://t.me/appleboom_sumy");
+
+        List<InlineKeyboardButton> row1 = getRow(menuButton);
+        List<InlineKeyboardButton> row2 = getRow(magazineButton);
+        List<InlineKeyboardButton> row3 = getRow(helpButton);
+        List<InlineKeyboardButton> row4 = getRow(contactButton);
+
+
+        List<List<InlineKeyboardButton>> rowList = getRowList(row1, row2, row3, row4);
+
+        return new InlineKeyboardMarkup(rowList);
+    }
+
+    public static InlineKeyboardMarkup NetAddressMenu() {
```

Comment:
# [ACROSS PR]

The function must be `netAddressMenu`, is not it?

PS: to be more exact `createNetAddressMenu` because this name lets you know the function creates a **new** keyboard by a certain pattern.